### PR TITLE
[RFC] Add utility constructors for the parser and use them in the service.

### PR DESCRIPTION
**Pull request by gfokkema**

### CODE REVIEW

File: src/main/java/nl/tudelft/dnainator/parser/buffered/DefaultEdgeParser.java [hunk 1]
```diff
@@ -26,6 +29,17 @@
 
 	/**
 	 * Constructs a {@link DefaultEdgeParser}, which reads from
+	 * the given {@link File}.
+	 * @param f	The {@link File} to read from.
+	 * @throws IOException	when file is not found or encoding is invalid
+	 */
+	public DefaultEdgeParser(File f) throws IOException {
+		this(new BufferedReader(new InputStreamReader(
+				new FileInputStream(f), "UTF-8")));
```
Comment (Balletie) [inline src/main/java/nl/tudelft/dnainator/parser/buffered/DefaultEdgeParser.java]: Do we know for sure that it's UTF-8?

File: src/main/java/nl/tudelft/dnainator/parser/buffered/DefaultEdgeParser.java [hunk 2]
```diff
@@ -26,6 +29,16 @@
 
 	/**
 	 * Constructs a {@link DefaultEdgeParser}, which reads from
+	 * the given {@link File}.
+	 * @param f	The {@link File} to read from.
+	 * @throws IOException	when file is not found or encoding is invalid
+	 */
+	public DefaultEdgeParser(File f) throws IOException {
+		this(new BufferedReader(new InputStreamReader(new FileInputStream(f), "UTF-8")));
```
Comment (Balletie) [inline src/main/java/nl/tudelft/dnainator/parser/buffered/DefaultEdgeParser.java]: How do we close the FileInputStream when we're done?

**Comment by gfokkema** [comment]
That is a really good question.
However, closing streams is apparently something we're already NOT doing...

This commit does not change that behaviour compared to upstream/master.

**Comment by Balletie** [comment]
Yes that is true, but then we had the to chance to close it. Can we still do it now?

**Comment by Unia** [comment]
Merged via #35

**Comment by Balletie** [comment]
No it's not, it says it has unmerged commits right here?

**Comment by Unia** [comment]
You're right, sorry :)

**Pull request closed without merging**